src/desktop/apps/partnerships/templates/common_section_mixins.jade

```diff
@@ -73,26 +69,3 @@ mixin apply(section)
     if path.indexOf('institution') > 0
```

Comment:
I guess this mixin is never called for 'auction-partnerships'. Wondering if we even need the conditional here?

Reply:
Looks like we don't! good thought